tensorflow/python/keras/benchmarks/keras_examples_benchmarks/antirectifier_benchmark_test.py

```diff
@@ -104,6 +104,46 @@ def benchmark_pixel_cnn_bs_512(self):
     self.report_benchmark(
         iters=run_iters, wall_time=wall_time, metrics=metrics, extras=extras)
 
+  def benchmark_anti_bs_512_gpu_1(self):
```

Comment:
Can you check the results of these new benchmark tests? In theory, as they use the same hyper-parameters, the performance number of gpu_2 should be 2x faster compared to gpu_1.

Reply:
I run these new benchmarks twice before and the `cifar10_cnn_benchmark_test` would be obvious, it achieves 80.67s and 51.41s in GPU_1, GPU2. And other models finish quickly and didn't show the twice performance as we wished. But, I can limit the GPU number of instance and run again.

Reply:
What are the performance of other models? 2x is a theoretical number. It might not be exactly 2x, but like cifar_cnn, it should be close.

What do you mean by "limit GPU number"? How does that affect performance?

Reply:
> What are the performance of other models?

You can check our doc `Results with PR`

> What do you mean by "limit GPU number"? How does that affect performance?

I mean, I can set the # of gpu in GCP as one or two.

I guess it will load the whole gpus and assign to specific tasks before, but I tried with this guess, and the results looks  similar, so, I think it's fine for it.

Reply:
# of GPUs on each instance should be no matter, as long as the number is not less than what you have requested in the dist-strat.

You may try to reduce the batch_size if the dataset is too small for the benchmarks.